subprojects/gradle/artifactory-app-backup/src/test/kotlin/com/avito/android/plugin/artifactory/ArtifactoryAppBackupPluginTest.kt

```diff
@@ -31,6 +31,7 @@ internal class ArtifactoryAppBackupPluginTest {
         TestProjectGenerator(
             modules = listOf(
                 AndroidAppModule(
+                    enableKotlinAndroidPlugin = false,
```

Comment:
boolean flags are evil

It'd be great to describe an intention by the configuration/DSL model.
Don't see how to express it. It'll require to rewrite API.

Out of scope of the PR

Reply:
duplicates https://github.com/avito-tech/avito-android/pull/586#discussion_r498214341